styleguide/_themes/derek/scss/snowflakes/events.scss

```diff
@@ -0,0 +1,364 @@
+  color: $white;
+  font-size: 2.5rem;
+  text-align: center;
+  text-transform: uppercase;
+}
+
+.eventsOverview-searchFilter-eventTypeSelect {
+  background: $white;
+  display: block;
+  margin: 1rem auto 0;
+  max-width: 200px;
+  width: $full-width;
+}
+
+.eventsOverview-searchFilter-eventTypeSubmit {
+  background: $gray-base;
+  border: 0;
+  color: $white;
+  display: block;
+  font-size: 1.2rem;
+  margin: 1rem auto 0;
+  max-width: 200px;
+  text-transform: uppercase;
+  width: $full-width;
+
+  &:hover {
+    background: $gray-dark;
+  }
+}
+
+.eventsOverview-searchResults {
+  @include make-sm-column(8);
+  @include make-md-column(9);
+  margin-bottom: 3rem;
+}
+
+.eventsOverview-weekEvent {
+  @include make-row;
+  background: $white;
+  border: 1px solid $gray-light;
+  margin: 0 0 .75rem;
+  text-align: center;
+}
+
+.eventsOverview-weekEvent-date {
+  background: $teal-dark;
+  color: $white;
+  font-weight: 600;
+  line-height: 3rem;
+  padding: 0 1.5rem;
+  text-transform: uppercase;
+}
+
+.eventsOverview-weekEvent-details {
+  padding: .5rem 1.5rem 1rem;
+}
+
+.eventsOverview-weekEvent-header {
+  @include make-row;
+}
+
+.eventsOverview-weekEvent-headerLocation {
+  @include make-xs-column(12);
+  @include make-md-column(5);
+  font-size: 1.2rem;
+  line-height: 4rem;
+  padding: 0 3rem 0 1.5rem;
+  text-transform: uppercase;
+}
+
+.eventsOverview-weekEvent-headerTitle {
+  @include make-xs-column(12);
+  @include make-md-column(7);
+  font-size: 2rem;
+  line-height: 4rem;
+  padding: 0 1.5rem 0 3rem;
+  text-transform: uppercase;
+}
+
+.eventsOverview-weekEvent-learnMore {
+  background: $white;
+  border: 1px solid $teal-dark;
+  display: block;
+  font-size: 1.3rem;
+  margin: 0 auto 1rem;
+  max-width: 90%;
+  padding: .75rem 2.25rem;
+  text-decoration: none;
+  text-transform: uppercase;
+
+  &:hover {
+    background: $teal-dark;
+    color: $white;
+    text-decoration: none;
+  }
+}
+
+.eventsOverview-weekTitle {
+  background: $gray-base;
+  color: $white;
+  font-size: 2.2rem;
+  margin-bottom: .75rem;
+  margin-top: 2.25rem;
+  padding: 1.5rem;
+  text-align: center;
+  text-transform: uppercase;
+
+  &:first-child {
+    margin-top: 0;
+  }
+}
+
+// Individual event page.
+.event {
+  @include make-container;
+}
+
+.event-backToEvents {
+  color: $teal-dark;
+  display: block;
+  margin-bottom: 1rem;
+  margin-left: $grid-gutter-width / 2;
+
+  &::before {
+    @include border-radius(50%);
+    background: $teal-dark;
+    border: 0;
+    color: $white;
+    content: '<';
+    margin-right: .5rem;
+    padding: 0 .5rem;
+    text-decoration: none;
+  }
+
+  &:hover {
+    color: $gray-darkest;
+
+    &::before {
+      background: $gray-darkest;
+      color: $white;
+      text-decoration: none;
+    }
+  }
+}
+
+.event-date {
+  @include make-sm-column(2);
+  margin-bottom: 1rem;
+}
+
+.event-dateBox {
+  background: $teal-dark;
+  color: $white;
+  font-family: $subhead;
+  font-size: 2.2rem;
```

Comment:
I dont love all the places we're defining font-size. When we change fonts its going to be hard to track all this down. Should we not just use one of our subhead declarations in the html instead of writing sass for all of this?